Turnstile-Ontology/02-Software/03-Implementation/.counter.d

```diff
@@ -0,0 +1 @@
+counter.exe: counter.c counter.h input.h hw.h output.h
```

Comment:
I believe these `.d` files shouldn't be committed.

Reply:
removed .d files